end-to-end-tests/src/helpers/ctx.rs

```diff
@@ -240,7 +240,7 @@ pub async fn build_client() -> Result<oxide_client::Client> {
             })
         },
         &Duration::from_secs(1),
-        &Duration::from_secs(300),
+        &Duration::from_secs(600),
```

Comment:
We're deploying two extra CockroachDB nodes and it can take a little longer for the system to set up now.

Reply:
I can look into parallelizing service bringup on the sled-agent side, if that would help mitigate?

Reply:
I expect it would.  No pressure on my part in terms of urgency.

Reply:
Sounds good, lets not block this PR, but I can do this as a follow-up.